src/book/03-guides/04-general/02-troubleshooting.mdx

```diff
@@ -61,9 +63,25 @@ find . -type f -name CMakeCache.txt -delete && find . -type d -name CMakeFiles -
 If the problem persists, try purging the build volume by running `./b configure --clean`.
 Note that purging the build volume means you'll have to rerun `./b build` to use built code.
 
+### Not enough memory in Docker? (Mac OS)
+
+By default the docker install will allocate 2GB of memory to docker images. When building the images this is insufficent for some of the libraries and will result in an error. In the menu bar go to advanced settings and increase the memory available.
+
 ## NUsight
 
-TODO
+### Multiple Docker robots in NUsight?
+
+This means someone else is also running a Docker robot. You can modify the name of your Docker robot by doing the following.
+
+```bash
+nano module/support/configuration/NetworkConfiguration/data/config/NetworkConfiguration.yaml
+```
```

Comment:
```suggestion

1. Open the `NetworkConfiguration.yaml` file in a text editor (here we are using nano on the command line):

   ```bash
   nano module/support/configuration/NetworkConfiguration/data/config/NetworkConfiguration.yaml
   ```
```

Reply:
```suggestion

1. Open the `NetworkConfiguration.yaml` file in a text editor (here we are using nano on the command line):
   ```bash
   nano module/support/configuration/NetworkConfiguration/data/config/NetworkConfiguration.yaml
   ```
```

Reply:
Looks like the ending ``` got cut off, good job GitHub 😆 
I'll commit and add it in manually.